src/Config.php

```diff
@@ -1438,6 +1438,24 @@ function formatGlobalLayout(layout) {
             echo "</td></tr>";
         }
 
+        echo "<tr class='tab_bg_1'><th colspan='4' class='center b'>" . _n('Notification', 'Notifications', Session::getPluralNumber()) . "</th></tr>";
+
+        echo "<tr class='tab_bg_2'>";
+        $content_helper = "<ul>
+            <li>" . __('Preselect "notification: no" in ITIL objects actor configuration, with ability to derogate to it') . "</li>
+            <li>" . __('Disable all notifications on all other objects, without ability to derogate to it.') . "</li>
+        </ul>
+        ";
+        $helper = "<span class='form-help' data-bs-toggle='popover' data-bs-placement='top' data-bs-html='true' data-bs-content='" . $content_helper . "'>?</span>";
```

Comment:
Not sure about this one, but content should probably be encoded. At least, if it contains double quotes, it will produce an unexpected attribute value.
```suggestion
        $helper = "<span class='form-help' data-bs-toggle='popover' data-bs-placement='top' data-bs-html='true' data-bs-content='" . htmlspecialchars($content_helper) . "'>?</span>";
```